packages/web3-eth/src/web3_eth.ts

```diff
@@ -308,7 +308,7 @@ export class Web3Eth extends Web3Context<Web3EthExecutionAPI, RegisteredSubscrip
 	 * ```
 	 */
 	public async calculateFeeData(
-		baseFeePerGasFactor = BigInt(2),
+		baseFeePerGasFactor: bigint | number = BigInt(2),
```

Comment:
Thanks @TemirlanBasitov I'll suggest to also add few examples and short description in above documentation comments so with this PR merge it will update docs website. https://docs.web3js.org/libdocs/Web3Eth#calculatefeedata

Reply:
@jdevcs  where i can add comment and short description here?

Reply:
I adjusted description of method with examples in the web3_eth.ts
Here also short explanation: 

calculateFeeData method now supports both bigint and number types for the baseFeePerGasFactor parameter. If a number is provided, it will be converted to a bigint with three decimal precision. This allows for more flexible calculations when dealing with decimal values.

Ex:
await calculateFeeData(BigInt(2));
// baseFeePerGasFactor remains as 2n

await calculateFeeData(1.5);
// baseFeePerGasFactor is converted to BigInt(1.500) for precise calculations

These changes make the method more versatile while ensuring backward compatibility with previous bigint usage.